tests/lib/rules/order-in-routes.js

```diff
@@ -361,6 +361,168 @@ eslintTester.run('order-in-routes', rule, {
+        queryParams: {},
+        currentUser: service(),
+        customProp: "test",
+        beforeModel() {},
+        model() {},
+        vehicle: alias("car"),
+        actions: {},
+        _customAction() {}
+      });`,
+      output: `export default Route.extend({
+        currentUser: service(),
+        queryParams: {},
+        customProp: "test",
+        vehicle: alias("car"),
+        beforeModel() {},
+        model() {},
+        actions: {},
+        _customAction() {}
+      });`,
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+      errors: [{
+        message: 'The "currentUser" service injection should be above the inherited "queryParams" property on line 2',
+        line: 3
+      }, {
+        message: 'The "vehicle" single-line function should be above the "beforeModel" lifecycle hook on line 5',
+        line: 7
+      }]
+    },
+    {
+      code: `export default Route.extend({
+        customProp: "test",
+        // queryParams line comment
+        queryParams: {},
+        model() {},
+        /**
+         * actions block comment
+         */
+        actions: {},
+        _customAction() {}
+      });`,
+      output: `export default Route.extend({
+        // queryParams line comment
+        queryParams: {},
+        customProp: "test",
+        model() {},
+        /**
+         * actions block comment
+         */
+        actions: {},
+        _customAction() {}
+      });`,
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+      errors: [{
+        message: 'The inherited "queryParams" property should be above the "customProp" property on line 2',
+        line: 4
+      }]
+    },
+    {
+      code: `export default Route.extend({
+        customProp: "test",
+        model() {},
+        /**
+         * beforeModel block comment
+         */
+        beforeModel() {},
+        /**
+         * actions block comment
+         */
+        actions: {},
+        _customAction() {}
+      });`,
+      output: `export default Route.extend({
+        customProp: "test",
+        /**
+         * beforeModel block comment
+         */
+        beforeModel() {},
+        model() {},
+        /**
+         * actions block comment
+         */
+        actions: {},
+        _customAction() {}
+      });`,
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+      errors: [{
+        message: 'The "beforeModel" lifecycle hook should be above the "model" hook on line 3',
+        line: 7
+      }]
+    },
+    {
+      code: `export default Route.extend({
+        customProp: "test",
+        /**
+         * actions block comment
+         */
+        actions: {},
+        /**
+         * beforeModel block comment
+         */
+        beforeModel() {}
+      });`,
+      output: `export default Route.extend({
+        customProp: "test",
+        /**
+         * beforeModel block comment
+         */
+        beforeModel() {},
+      /**
```

Comment:
Ok, so the indentation is wrong here as well, but `[indent]` rule doesn't fix this, so I have to think how to prevent that from happening.